controller/argument_value_resolver.rst

```diff
@@ -146,10 +150,10 @@ and adding a priority.
 
         # app/config/services.yml
         services:
-            app.value_resolver.user:
-                class: AppBundle\ArgumentResolver\UserValueResolver
-                arguments:
-                    - '@security.token_storage'
+            # ...
+
+            AppBundle\ArgumentResolver\UserValueResolver:
+                # arguments is not needed if you have autowire above under _defaults
```

Comment:
I don't mind putting it everywhere :). I just want the absolute best message. My last commit - sha: ab4de028c0464a96def6cc9baa45382d64802e50 - gives a uniform handling of this (for the files in this PR). 
The big risk of confusion now is people thinking that they need to add `_defaults` to their file (e.g. so they have 2 `_defaults`) or make their `_defaults` look exactly like our's (so, remove `public` and `autoconfigure`). I'm interested in using the `diff` type for this (there's not a *ton* of highlighting), but I really want #7806 first (if my new comment is possible, it would be AMAZING!). I haven't seen @wouterj in a bit though... I'm trying to ping and bother him :)